tests/support/holdings.py

```
@@ -0,0 +1,111 @@
+
+    @property
+    def available(self):
+        """True when a real holdings tree was resolved (flavor is 'full' or 'mini')."""
+        return self.flavor is not None
+
+
+def _skip_config():
+    return HoldingsConfig(None, _PLACEHOLDER_PDS3, _PLACEHOLDER_PDS4)
+
+
+def _resolve_full():
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_resolve_full`

(ANN202)